README.md

```diff
@@ -1,21 +1,34 @@
 # Swift for Visual Studio Code
 
-## Installation
+This extension adds language support for Swift to Visual Studio Code. It supports:
 
-Currently, the only way to install this extension is to build it from source. To do this, you'll need to have [Node.js](https://nodejs.org) installed. On Linux, make sure to install Node.js from its official website or from [NodeSource](https://github.com/nodesource/distributions/) as the version included with your distribution may be outdated.
+* Automatic task creation
+* Package dependency view
+* Code completion
+* Jump to definition, peek definition, find all references, symbol search
+* Error annotations and apply suggestions from errors
```

Comment:
Maybe include
* Automatic generation of launch configurations for debugging with [CoreLLDB](https://marketplace.visualstudio.com/items?itemName=vadimcn.vscode-lldb).